specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -2706,8 +2706,8 @@
           "description": "This value specifies the endpoint to access with the S3 Connector. This is an optional property; change it only if you want to try a different service endpoint or want to switch between https and http. Type: string (or Expression with resultType string)."
         },
         "sessionToken": {
-          "type": "object",
-          "description": "The session token for the S3 temporary security credential. Type: string (or Expression with resultType string)."
+          "$ref": "../datafactory.json#/definitions/SecretBase",
```

Comment:
This mistake was added by last PR https://github.com/Azure/azure-rest-api-specs/pull/10903 just now. And there's no SDK release after that.
So it won't be a breaking change.